chain/network/src/types.rs

```diff
@@ -1272,6 +1292,34 @@ pub enum NetworkRequests {
 
     /// A challenge to invalidate a block.
     Challenge(Challenge),
+
+    // IbfMessage
+    IbfMessage {
+        peer_id: PeerId,
+        ibf_msg: RoutingSyncV2,
+        ibf_wrapper: IbfWrapper,
+    },
+}
+
+#[derive(Clone)]
+pub struct IbfWrapper {
+    pub ibf_set: Arc<Mutex<IbfSet<SimpleEdge>>>,
+}
+
+impl Debug for IbfWrapper {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_tuple("").finish()
+    }
+}
+
+impl Eq for IbfWrapper {
+    fn assert_receiver_is_total_eq(&self) {}
+}
```

Comment:
```suggestion
impl Eq for IbfWrapper {
}
```

That's a private impl detail of std, it shouldn't  be manually implemented.